src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Text.cs

```diff
@@ -34,7 +36,7 @@ public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
             yield return new[] { TexlStrings.TextArg1, TexlStrings.TextArg2, TexlStrings.TextArg3 };
         }
 
-        public override bool CheckTypes(TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
+        public override bool CheckTypes(CheckTypesContext checkTypesContext, TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
```

Comment:
>args [](http://example.com/codeflow?start=88&length=4)

One possible argument is ResultLanguageTag (CultureInfo name). Should ResultLanguageTag argument get validated to check if is a valid one?

Reply:
The check for the third argument should be done at runtime